README.md

```diff
@@ -1 +1,66 @@
-# Forms-html
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: Mint Sign Up page clone
+
+> In this project we worked with forms and inputs in HTML and was styled with CSS3, to create a sign up page clone of Intuit Mint.
+
+![screenshot](./graphics/Mint-Clone-screenshot.png)
+
+
+## Built With
+
+- HTML, CSS3
+- No frameworks used
+- VS code with live server
+
+## Live Demo
+
+[Live Demo Link](https://sanadwj.github.io/Forms-html/)
```

Comment:
fix the live demo link it's not working you can you [githack](https://raw.githack.com/)